.github/workflows/release-snapshot.yml

```diff
@@ -0,0 +1,38 @@
+name: Release snapshot
+
+# Releases a snapshot release when new commits merge to develop
+# This ensures the release process is working as expected as well as always gives us a release
+# we should move to a single branch in near future
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+
+jobs:
+  snapshot-snapshot:
+    name: Publish snapshot release to npm
+    if: github.repository == 'eth-optimism/optimism'
```

Comment:
This makes it so this workflow doesn't run on forks of optimism

Reply:
Don't you mean ethereum-optimism?

Reply:
```suggestion
    if: github.repository == 'ethereum-optimism/optimism'
```